debug-writer/README.md

```diff
@@ -0,0 +1,4 @@
+Compose Desktop Application
+
+- `./gradlew run` - run application
```

Comment:
When I run on Windows:
![image](https://user-images.githubusercontent.com/5963351/117314433-6ef46980-ae8f-11eb-84ff-f3a09a17f90c.png)

Looking at the code it seems this project is just a template and we will add functionality in the future?